docs/tutorial/08-marketplace-compose.md

```diff
@@ -681,10 +705,13 @@ transaction {
     prepare(acct: auth(Capabilities, BorrowValue) &Account) {
```

Comment:
I don't think this transaction needs any `Capabilities` access, looks like it's just borrowing from storage and `.get()` is used to access a public cap

```suggestion
    prepare(acct: auth(BorrowValue) &Account) {
```